packages/server-admin-ui/src/views/appstore/appStore.scss

```
@@ -238,18 +238,19 @@
 .plugin-card .card-body,
 .plugin-card .card-footer {
   position: relative;
-  // Keep the visible content above the overlay so text selection and
-  // hover styles still work; the link still receives clicks because
-  // these elements don't intercept them on their own.
-  z-index: 0;
+  // No z-index here: a z-index would create a stacking context that
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)